sphinx/ext/apidoc.py

```diff
@@ -45,7 +46,7 @@
     ]
 
 INITPY = '__init__.py'
-PY_SUFFIXES = {'.py', '.pyx'}
+PY_SUFFIXES = ('.py', '.pyx', sysconfig.get_config_var('EXT_SUFFIX'))
```

Comment:
It seems `importlib.machinery.EXTENSION_SUFFIXES` is much better than this.
https://docs.python.org/3/library/importlib.html#importlib.machinery.EXTENSION_SUFFIXES